### ISSUE TITLE
not creating custom sensors, battery, total cutting time, etc

### ISSUE THREAD

**Issue by andyb2000** [andyb2000/HACS-husqvarna_automower_ble#1]
Currently a bug that does not add the additional sensors from sensor.py

Logs show that it is entering the sensor creation loop but when async_add_entities is called it has blank data.

Logs:

```
2024-07-11 10:57:44.208 DEBUG (MainThread) [custom_components.husqvarna_automower_ble.sensor] Creating mower sensors
2024-07-11 10:57:44.208 DEBUG (MainThread) [custom_components.husqvarna_automower_ble.sensor] in AutomowerSensorEntity creating entity for: Battery Level
2024-07-11 10:57:44.208 DEBUG (MainThread) [custom_components.husqvarna_automower_ble.sensor] in AutomowerSensorEntity creating entity for: Next Start Time
2024-07-11 10:57:44.208 DEBUG (MainThread) [custom_components.husqvarna_automower_ble.sensor] in AutomowerSensorEntity creating entity for: Total running time
2024-07-11 10:57:44.208 DEBUG (MainThread) [custom_components.husqvarna_automower_ble.sensor] in AutomowerSensorEntity creating entity for: Total cutting time
2024-07-11 10:57:44.208 DEBUG (MainThread) [custom_components.husqvarna_automower_ble.sensor] in AutomowerSensorEntity creating entity for: Total charging time
2024-07-11 10:57:44.208 DEBUG (MainThread) [custom_components.husqvarna_automower_ble.sensor] in AutomowerSensorEntity creating entity for: Total searcing time
2024-07-11 10:57:44.208 DEBUG (MainThread) [custom_components.husqvarna_automower_ble.sensor] in AutomowerSensorEntity creating entity for: Total number of collisions
2024-07-11 10:57:44.208 DEBUG (MainThread) [custom_components.husqvarna_automower_ble.sensor] in AutomowerSensorEntity creating entity for: Total number of charging cycles
2024-07-11 10:57:44.208 DEBUG (MainThread) [custom_components.husqvarna_automower_ble.sensor] in AutomowerSensorEntity creating entity for: Total cutting blade usage
2024-07-11 10:57:44.209 DEBUG (MainThread) [custom_components.husqvarna_automower_ble.sensor] About to add sensors: [<entity unknown.unknown=unknown>, <entity unknown.unknown=unknown>, <entity unknown.unknown=unknown>, <entity unknown.unknown=unknown>, <entity unknown.unknown=unknown>, <entity unknown.unknown=unknown>, <entity unknown.unknown=unknown>, <entity unknown.unknown=unknown>, <entity unknown.unknown=unknown>]
2024-07-11 10:57:44.209 DEBUG (MainThread) [custom_components.husqvarna_automower_ble.sensor] state of sensor data structure: {'battery_level': 100, 'activity': <MowerActivity.PARKED: 5>, 'state': <MowerState.RESTRICTED: 7>, 'next_start_time': datetime.datetime(2024, 7, 11, 12, 0, tzinfo=datetime.timezone.utc), 'statistics': {'totalRunningTime': 2646768, 'totalCuttingTime': 2511938, 'totalChargingTime': 812706, 'totalSearchingTime': 121312, 'numberOfCollisions': 12667, 'numberOfChargingCycles': 313, 'cuttingBladeUsageTime': 2511938}, 'operatorstate': 0}

```

**Comment by andyb2000** [comment]
Potentially resolved in 96d11c2

**Comment by andyb2000** [comment]
Closing as dev merged to main
